src/applications/mhv/medications/containers/Prescriptions.jsx

```diff
@@ -308,11 +309,20 @@ const Prescriptions = () => {
             buildPrescriptionsTXT(prescriptionsFullList),
             buildAllergiesTXT(allergies),
           );
+        } else {
+          updateLoadingStatus(false, '');
         }
+      } else if (
+        prescriptionsFullList?.length &&
+        allergiesError &&
+        pdfTxtGenerateStatus.status === PDF_TXT_GENERATE_STATUS.InProgress
+      ) {
+        updateLoadingStatus(false, '');
```

Comment:
I think this if else condition could be simplified (i.e. to not write `updateLoadingStatus(false, '');` twice